$NetBSD: patch-jna_build.xml,v 1.1 2018/05/20 12:53:01 fhajny Exp $

Skip API check, we build from source anyway. Removes silly dependency on ggrep.

--- jna/build.xml.orig	2017-12-27 19:27:16.000000000 +0000
+++ jna/build.xml
@@ -682,7 +682,7 @@ osname=macosx;processor=x86;processor=x8
     </condition>
     <!-- args are based on GNU grep, other versions may differ -->
     <apply dir="${build.native}" executable="${grep}" parallel="true"
-           failonerror="${grep.required}" relative="true" output="${md5.file}">
+           relative="true" output="${md5.file}">
       <arg value="-A"/>
       <arg value="1"/>
       <arg value="JNIEXPORT"/>
@@ -790,7 +790,7 @@ osname=macosx;processor=x86;processor=x8
                    file="${rsrc}" byline="true"/>
   </target>
 
-  <target name="native" depends="-enable-native,-setup,javah,-native-api-check,rsrc" unless="-native"
+  <target name="native" depends="-enable-native,-setup,javah,rsrc" unless="-native"
           description="Build native libraries.  Use 'ant -DCC=xxx' to build using a compiler other than gcc">
     <property name="comment" value="# auto-generated by ant"/>
     <replaceregexp match="^JNA_JNI_VERSION=.*"
